### DISCUSSION TITLE
Easy Contribution

### DISCUSSION THREAD

**Discussion by Astrodevil** [Astrodevil/Programming-Basics#21] [Q&A] [answered]
This Repository contains basic level programming language source codes. Any newbie learning any language can contribute here. 

**Raise a New Issue So that I can assign it you**

**Comment by subh-chaturvedi** [comment]
I am new to open source contribution and sorry if this is not the right question to ask, but contributing to this repo will be considered for Hacktober Fest 2021, right?

**Comment by Astrodevil** [comment, marked as answer]
Yes, after the start of the program it will be counted. Till then you can practice

**Comment by codin-champ** [comment]
Hi I wish to start contributing to Open Source. What can I work on in this repository?

**Comment by Astrodevil** [comment]
> 
> 
> Hi I wish to start contributing to Open Source. What can I work on in this repository?

As you can see this repository contains codes related to various programming languages and DSA. You can also contribute the same. Just put your codes in right folder.
